db/seeds.d/106-mail_notifications.rb

```diff
@@ -30,7 +30,7 @@
 ]
 
 notifications.each do |notification|
-  ::MailNotification.find_or_create_by_name(notification)
+  ::MailNotification.create(notifications) unless ::MailNotification.where(:name => notification[:name]).exists?
```

Comment:
I know this doesn't seem like rails convention here, but I had a discussion with @parthaa about this, he suggested that I just check for name in case someone has change the attributes manually, that way the changes would stay when seeding. ```::MailNotification.where(name: notification[:name]).first_or_create``` wouldn't update with the other attributes in the notification hash. If we added the other attributes in the where query, then user changes wouldn't stay